src/NHibernate/Logging.log4net.cs

```diff
@@ -12,12 +12,13 @@ namespace NHibernate
 	public class Log4NetLoggerFactory: ILoggerFactory
 #pragma warning restore 618
 	{
-		private static readonly System.Type LogManagerType = System.Type.GetType("log4net.LogManager, log4net");
+		private static readonly System.Type LogManagerType;
 		private static readonly Func<Assembly, string, object> GetLoggerByNameDelegate;
 		private static readonly Func<System.Type, object> GetLoggerByTypeDelegate;
 
 		static Log4NetLoggerFactory()
 		{
+			LogManagerType = System.Type.GetType("log4net.LogManager, log4net", true);
 			GetLoggerByNameDelegate = GetGetLoggerByNameMethodCall();
 			GetLoggerByTypeDelegate = GetGetLoggerMethodCall<System.Type>();
```

Comment:
The method calls following the get type are using the type without checking if it was found, so better not allow `GetType` to yield `null`.